common/openstack/adding-an-openstack-cloud-provider.adoc

```diff
@@ -73,6 +73,8 @@ In environments that use Keystone v3 authentication, the user must have the *adm
 
 [NOTE]
 ====
-To collect inventory and metrics from an OpenStack environment, the {product-title} appliance requires that the adminURL endpoint for the OpenStack environment be on a non-private network.
+* To collect inventory and metrics from an OpenStack environment, the {product-title} appliance requires that the adminURL endpoint for the OpenStack environment be on a non-private network.
 Hence, the OpenStack adminURL endpoint should be assigned an IP address other than `192.168.x.x`. Additionally, all the Keystone endpoints must be accessible, otherwise refresh will fail.
+* Collecting C & U data from am OpenStack cloud provider requires selecting the *Collect for All Clusters* option under *Configuration*, in the setting menu. For information, see  https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.5/html-single/general_configuration/#enabling-a-cluster,-host,-or-datastore-for-capacity-and-utilization-collection[ Enabling a Cluster, Host, or Datastore for Capacity and Utilization Collection] in the Red Hat CloudForms _General Configuration_ guide.
```

Comment:
A few fixes needed for this sentence: "Collecting C & U data from am OpenStack cloud provider requires selecting the *Collect for All Clusters* option under *Configuration*, in the setting menu."
- am --> an
- C & U data --> I'd recommend spelling out as "capacity and utilization data", just to avoid acronyms. Looking through the guide we only abbreviate it when it's a UI label.
- setting menu --> settings menu
- The Gen Config link currently goes to the top of the guide -- I know there's not a great way to link to sections in this behemoth guide, but would it be better to link to 4.1.1 Regions maybe? https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.5/html-single/general_configuration/#regions That's how I tend to handle links to the GC guide, but happy to discuss ideas. 
- "Red Hat CloudForms" should either be an entity, or could be removed for upstreaming.